src/Icons/svgr/IconAvatar.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import React from 'react';
```

Comment:
We need import all for flow type checking

```suggestion
import * as React from 'react';
```